pom.xml

```diff
@@ -1,391 +1,421 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
can you do a commit where only the diffs introduced by the PR are shown? without the IDE reformat?

Reply:
Fixed!!